mh/session.py

```diff
@@ -297,6 +299,21 @@ def try_transfer_city_leadership(self):
             return player
         return None
 
+    def try_transfer_circle_leadership(self):
+        if self.local_info['circle_id'] is None:
+            return None
+
+        circle = self.get_circle()
+        if not (circle.departed and circle.leader == self and circle.get_population() > 1):
```

Comment:
What's the difference?

Reply:
Less long, faster and more explicit. I don't have to read the whole condition and invert it to know what its purpose is.

Reply:
I find it more explicit this way, as it describes the one single state in which this function should ever do anything (on a quest, you're the host, AND there's at least one other player). I'll still change it since it's your call.